swift-sdk/Internal/EmbeddedSessionManager.swift

```diff
@@ -62,7 +62,7 @@ public class EmbeddedSessionManager {
         }
     }
 
-    public func startImpression(messageId: String) {
+    public func startImpression(messageId: String, placementId: Int64) {
```

Comment:
Int64 why? I think it will have be a string
@mprew97 what say?

Reply:
Updating to Double.